CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 ##### Breaking
 
-* None.
+* Now `libclang.dylib` and `sourcekitd.framework` are dynamicaly loaded on
+  runtime by `SourceKittenFramework`. Depending client might need some
+  modification if referencing them from language other than Swift.  
+  [Norio Nomura](https://github.com/norio-nomura)
+  [#167](https://github.com/jpsim/SourceKitten/issues/167)
```

Comment:
If you at call `dlopen("libclang.dylib", …)` or `dlopen("sourcekit.framework/sourcekitd", …)` at some point, the normal paths will be searched. This would allow something that works by setting `DYLD_LIBRARY_PATH` and `DYLD_FRAMEWORK_PATH` (like Jazzy) to continue to work with no changes. This would require some changes to the code that's there right now, such as handling the error from `dlopen`.